docs/adapters/development.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+permalink: /docs/adapters/development/index.html
+---
+
+# Adapter Basics
+
+All adapters inherit from the Adapter class in the `src/adapter.coffee` file.  There are certain methods that you will want to override.  Here is a basic stub of what an extended Adapter class would look like:
+
+```coffee
+class Bunny extends Adapter
+
+  constructor: ->
+    @robot.logger.info "Constructor"
+    super
+    
+
+  send: (envelope, strings...) ->
+    @robot.logger.info "Send"
+
+  reply: (envelope, strings...) ->
+    @robot.logger.info "Reply"
+
+  run: ->
+    @robot.logger.info "Run"
+    @emit "connected"
+
+
+exports.use = (robot) ->
+  new Bunny robot
+```
+
+# Setting Up Your Development Environment
+
+1. Create a new folder for your adapter `hubot-bunny`
+  - `mkdir hubot-bunny`
+2. Change your working directory to `hubot-bunny`
+  - `cd hubot-bunny`
+3. Run `npm init` to create your package.json
+  - make sure the entry point is `src/bunny.coffee`
+4. Add your `.gitignore` to include `node_modules`
+5. Edit the `src/bunny.coffee` file to include the above stub for your adapter
+6. Edit the `package.json` to add a peer dependency on `hubot`
+
+  ```json
+  "dependencies": {
+  },
+  "peerDependencies": {
+    "hubot": "^2.11.1"
```

Comment:
Should the hubot peer dependency have a pegged version number, or simply `"hubot": ""'